### DISCUSSION TITLE
vant2 怎么能设置全局默认属性，比如按钮的大小。

### DISCUSSION THREAD

**Discussion by bbhxwl** [youzan/vant#10509] [General]
vant2 怎么能设置全局默认属性，比如按钮的大小。

**Comment by cloudmoonocus** [comment]
多看文档描述
![image](https://user-images.githubusercontent.com/98140702/183251100-310a1234-d550-4ad6-9f20-fe4b3e9f4d58.png)

https://vant-contrib.gitee.io/vant/v2/#/zh-CN/theme
